src/kv/test/stub_consensus.h

```diff
@@ -11,17 +11,22 @@
 
 namespace kv
 {
+  static NodeId PrimaryNodeId = std::string("PrimaryNodeId");
+  static NodeId FirstBackupNodeId = std::string("FirstBackupNodeId");
+  static NodeId SecondBackupNodeId = std::string("SecondBackupNodeId");
+  static NodeId ThirdBackupNodeId = std::string("ThirdBackupNodeId");
```

Comment:
If these are just used for testing, they shouldn't be in the `kv` namespace. That's probably true of the existing things in this namespace too, but `kv::PrimaryNodeId` is more obviously something which shouldn't exist.

Reply:
Renamed this namespace from `kv` to `kv::test`/